charts/pega/README.md

```diff
@@ -208,7 +208,18 @@ Example:
 requestor:
   passivationTimeSec: 900
 ```
+### Security context
 
+You may configure security context for Pega pod deployments. By default ```pegasystems/pega``` image uses ```pegauser(9001)``` as the user.
```

Comment:
By default, to configure a security context for your Pega pod deployments the ```pegasystems/pega``` image uses ```pegauser(9001)``` as the user. to configure an alternative user, do this___________, whatever it is.

Reply:
you can address these documentation changes later using BUG-622868. All set!